src/lib/holocene/menu/menu.svelte

```diff
@@ -12,15 +12,16 @@
   interface $$Props extends HTMLAttributes<HTMLUListElement> {
     id: string;
     keepOpen?: boolean;
-    position?: 'left' | 'right';
+    position?: 'left' | 'right' | 'topLeft' | 'topRight';
     menuElement?: HTMLUListElement;
   }
 
   let className = '';
+  let height = 0;
   export { className as class };
   export let id: string;
   export let keepOpen = false;
-  export let position: 'left' | 'right' = 'left';
+  export let position: 'left' | 'right' | 'topLeft' | 'topRight' = 'left';
```

Comment:
can you make this `kebab-case`? I hate `camelCase` string props, when they're kebab-case, they read more like regular html attributes

Reply:
👍